Support "local" as value for EXAQUTE_BACKEND

quentin-ag opened the pull request:
# Motivation 

Currently, the only way to select the local backend is to unset the `EXAQUTE_BACKEND` variable in the environment.

I would like to be able to choose explictly the local backend with the value I set to `EXAQUTE_BACKEND`, not just as a fallback option. It is not necessary but I find it more intuitive, and it makes the experience easier in many small ways (e.g. shell scripting, checking the environment, etc.).

# Proposed change

`"local"` is an accepted value for `EXAQUTE_BACKEND`. As before, the message printed is slightly different depending on whether the backend was chosen explicitly or set by default.

Also, the error messages for other values of `EXAQUTE_BACKEND` (supported or not) are a bit more informative.

Feel free to edit, suggest, comment, etc.

Commit:
add(init): support "local" as value for EXAQUTE_BACKEND

Also: add more helpful error message for other (non-supported) values.

riccardotosi commented on exaqute/__init__.py:
```diff
@@ -2,18 +2,21 @@
-    print("ExaQUte backend: Local")
+if exaqute_backend == "local":
     from .local import *  # noqa
-elif exacute_backend == "hyperloom":
-    from .hyperloom import *  # noqa
-elif exacute_backend == "pycompss":
-    from .pycompss import *  # noqa
```
Why are you removing `hyperloom` and `pycompss` imports?

jorgee replied:
these lines shoudn't be removed.

quentin-ag replied:
Indeed, that's a stupid mistake. This removal was copied from the `exaqute` module in XMC, which has only the `local` backend. I had replaced these with an error message.  Thanks for catching this, Riccardo!

I have fixed it in commit `17c5ede332aecf3109a08ea8526003bb8f652fb0`.

Commit:
fix(init): restore support for hyperloom and pycompss backend

Removed by error in commit 0fc69920633ba3420e34c7dcc7e6fd678ac7d5d7.
List supported values in error message.
